app/helpers/application_helper.rb

```diff
@@ -6,17 +6,12 @@ def flash_class(level)
 
   def markdown(text)
     options = {
-      filter_html:     true,
-      hard_wrap:       true,
-      link_attributes: { rel: 'nofollow', target: "_blank" },
-      space_after_headers: true,
-      fenced_code_blocks: true
+      filter_html: true, hard_wrap: true, link_attributes: { rel: 'nofollow', target: "_blank" },
```

Comment:
Line is too long. [97/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.